security/src/main/java/uk/ac/ox/ndph/mts/security/authorisation/DenyMethodSecurityConfig.java

```diff
@@ -17,11 +16,12 @@
 public class DenyMethodSecurityConfig extends GlobalMethodSecurityConfiguration { //NOSONAR
```

Comment:
Should this not be enabled for this PR? Unless I'm misreading the comment.

Reply:
We agreed to disable that validation. It requires that on each endpoint we have authorisation annotation, currently we added it just to one endpoint - create person, once we have fully integrated auth we can return this validation... It will prevent introducing new endpoints that do not have authorisation annotation on them.